Dockerfile

```diff
@@ -103,13 +103,13 @@ RUN set -ex\
 	;
 
 # Config-tool builds the go binary in the configtool.
-FROM registry.access.redhat.com/ubi8/go-toolset:1.16.12 as config-tool
+FROM registry.access.redhat.com/ubi8/go-toolset:1.17.12 as config-tool
 WORKDIR /opt/app-root/src
 ARG CONFIGTOOL_VERSION=v0.1.12
 RUN curl -fsSL "https://github.com/quay/config-tool/archive/${CONFIGTOOL_VERSION}.tar.gz"\
 	| tar xz --strip-components=1 --exclude '*/pkg/lib/editor/static/build'
 COPY --from=config-editor /opt/app-root/src/static/build  /opt/app-root/src/pkg/lib/editor/static/build
-RUN go install ./cmd/config-tool
+RUN go install -tags=fips ./cmd/config-tool
```

Comment:
Where does this tag come from?

Reply:
See https://github.com/minio/minio-go/pull/1700 and this comment https://github.com/minio/minio-go/issues/1697#issuecomment-1259523447

Reply:
LGTM